lambdaclass/mirra_backend
# [GH-591] Allow movement while executing skills

## Motivation

Some skills should be able to be cast while moving

Merge alongside [This](https://github.com/lambdaclass/curse_of_mirra/pull/1784) client pr

Closes #591

### Disclaimer
Even tho you can move while you execute a skill and we don't stop this in the backend the unity client still blocks the inputs in the game while executing the skill, this will be addressed in a future PR in the client repo. The game configuration will be the same 

## Summary of changes

Refactored the block of actions while executing actions to block movement depending on a configuration

## How to test it?

Change any skill `block_movement` configuration to false, and you should keep moving while this action is being executed. This pr is meant to be used for the h4ck basic but you should be able to change other abilities 

Start a match and use the h4ck basic and you should be able to move while shooting

## Checklist
